doc/ModuleStartupOrder.md

```diff
@@ -116,7 +116,9 @@ The following sample deployment manifest illustrates how startupOrder values of
 
 In the sample deployment manifest shown above:
 
-* The `$edgeHub` module has been assigned a `startupOrder` value of 0.
+* The '$edgeAgent' schemaVersion has been set to 1.1 (or later).
+* The 'edgeAgent' module's image is 1.0.10 (or later).
+* The `edgeHub` module has been assigned a `startupOrder` value of 0.
```

Comment:
@richma-ms The `startupOrder` of `edgeAgent` is also 0, right? We can make an explicit comment here as well.

Reply:
There's a note given in an earlier section of the doc describing the behavior for edgeAgent.  I'll reiterate it here as well.